core/src/main/java/app/majime/core/sample/Sample.java

```diff
@@ -19,20 +19,23 @@
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sample_seq")
     private Long id;
 
-    @NonNull
-    @Column(name = "sample_no")
+    @Column(name = "sample_no", length = 50, nullable = false)
     private String sampleNo;
 
+    @Column(nullable = false)
     private int quantity;
 
-    private int status;
+    @Column(nullable = false)
+    private Long status;
 
-    private int batch_id;
+    @Column(nullable = false)
+    private Long batch_id;
 
-    private int user_id;
+    @Column(nullable = false)
```

Comment:
```suggestion
    @NotNull
```
It's deprecated!! Plz Boy, check it out